src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/RazorEditorFactory.cs

```diff
@@ -0,0 +1,166 @@
+﻿using System;
```

Comment:
Copyright. On other files on this PR as well.

Reply:
Working on aspnetcore-tooling makes me appreciate the linting we do in the TS-world and wish for it in CS-land.

Reply:
It's possible to use stylecop to enforce the copyright header (and other else). Feel free